test/gpu/literal.cpp

```diff
@@ -48,4 +48,4 @@ void gpu_literal_test()
     }
 }
 
-int main() { gpu_literal_test(); }
+int main() { gpu_literal_test(); } // NOLINT
```

Comment:
What is the lint error here?

Reply:
[ 60%] clang-tidy: Running clang-tidy on target /workspace/AMDMIGraphX/test/onnx/onnx_rnn_test.cpp...
62310 warnings generated.
::warning file=test/gpu/literal.cpp,line=51,col=5::an exception may be thrown in function 'main' which should not throw exceptions [bugprone-exception-escape,-warnings-as-errors]
int main() { gpu_literal_test(); }
    ^
note: expanded from here

CMake Error at tidy-target-test_gpu_literal-_workspace_AMDMIGraphX_test_gpu_literal_cpp.cmake:80 (message):
  Clang tidy failed.

Reply:
Can you do `NOLINT(bugprone-exception-escape)` to make it clearer which warnings you are trying to suppress?